include/aca_oam_server.h

```diff
@@ -55,7 +55,7 @@ struct flow_inject_msg {
   uint16_t src_port; // Inner Packet SPort
   uint16_t dst_port; // Inner Packet DPort
   uint8_t proto; // Inner Packet Protocol
-  uint8_t vni[4]; // Inner Packet Protocol
+  uint8_t vni[3]; // Inner Packet Protocol
```

Comment:
why 3? please add comment.

Reply:
My comment here is wrong, sorry. In oam packet, vni has 24 bits, occupying three bytes.